docs/components/Buttons.md

```diff
@@ -125,17 +132,24 @@ OdsTextButton(
 )
 ```
 
+In the same way as for other buttons, you can display an `OdsTextButton` in the inverse theme of the default system theme by putting it into an `InverseTheme` composable:
+
+```kotlin
+InverseTheme {
+    // Put here the `OdsTextButton` you want to display in inverse theme
+}
+```
+
```

Comment:
I wonder if we should still talk about this here because `InverseTheme` will work with all components and is not specific to buttons.